benchmark_runner/common/oc/oc.py

```diff
@@ -227,8 +227,8 @@ def _get_pod_name(self, pod_name: str,
         except Exception as err:
             raise PodNameNotExist(pod_name=pod_name)
 
-    def _is_pod_exist(self, pod_name: str,
-                      namespace: str = environment_variables.environment_variables_dict['namespace']):
+    def is_pod_exist(self, pod_name: str,
```

Comment:
If you're going to change the name, make it correct: `does_pod_exist` or `pod_exists`

Reply:
I will change it to pod_exists